### PR TITLE
chore: renamed boilerplate leftovers and edited the openapi

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: openapi3.yaml [hunk 1]
```diff
@@ -1,87 +1,79 @@
...
-        id:
-          type: number
-          format: int64
-        name:
-          type: string
-        description:
-          type: string
-    anotherResource:
+
+    tilesRequest:
```
Comment (vitaligi) [inline openapi3.yaml]: `TilesRequest` use pascal case

File: openapi3.yaml [hunk 2]
```diff
@@ -1,87 +1,79 @@
...
-        - isAlive
+        - bbox
+        - minZoom
+        - maxZoom
       properties:
-        kind:
-          type: string
-        isAlive:
-          type: boolean
+        bbox:
```
Comment (vitaligi) [inline openapi3.yaml:65]: add an example for `bbox`, currently the example shows an invalid `bbox` [0,0,0,0]

File: openapi3.yaml [hunk 3]
```diff
@@ -1,87 +1,79 @@
...
+          minItems: 4
+          maxItems: 4
+        minZoom:
+          type: integer
+          minimum: 0
+          maximum: 18
+        maxZoom:
+          type: integer
+          minimum: 0
+          maximum: 18
```
Comment (vitaligi) [inline openapi3.yaml:81]: should we limit it to 18 or some larger value, the same for `minZoom`
perhaps add a description indicating `maxZoom` must be equal to at least `minZoom`

**Review by vitaligi — CHANGES_REQUESTED** [review]

File: openapi3.yaml (hunk 3 above)
Comment (CptSchnitz) [reply openapi3.yaml:81]: in regards to the max value of the zoom, i think later we can make it configurable.

File: openapi3.yaml (hunk 1 above)
Comment (CptSchnitz) [reply openapi3.yaml]: fixed

File: openapi3.yaml (hunk 2 above)
Comment (CptSchnitz) [reply openapi3.yaml:65]: added

File: openapi3.yaml (hunk 3 above)
Comment (CptSchnitz) [reply openapi3.yaml:81]: Added better description

**Review by vitaligi — APPROVED** [review]

**Pull request merged**